Tests/GRPCTests/GRPCTestCase.swift

```diff
@@ -46,6 +65,14 @@ class GRPCTestCase: XCTestCase {
     super.setUp()
     XCTAssertTrue(GRPCTestCase.isLoggingConfigured)
   }
+
+  var runTimeSensitiveTests: Bool {
+    let shouldRun = GRPCTestCase.runTimeSensitiveTests
+    if !shouldRun {
+      print("Skipping '\(self.name)' as ENABLE_TIMING_TESTS=false")
```

Comment:
This side effect feels like a code small; at least convert this into a `func`? Also, what will `self.name` evaluate to?

Reply:
Sure, let's make it a func. I'm okay with the side effect as it's only a print message in tests.

`self.name` looks like "-[FunctionalTestsMutualAuthentication testUnaryLotsOfRequests]"